db/migrate/20250210071551_expand_cancellation_restrictions_for_competitions.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ExpandCancellationRestrictionsForCompetitions < ActiveRecord::Migration[7.2]
+  def up
+    add_column :Competitions, :cancellation_restrictions, :integer, default: 0, null: false
+    Competition.where(allow_registration_self_delete_after_acceptance: false)
+               .update_all(cancellation_restrictions: 1)
```

Comment:
You can use the enum value literal here.

Reply:
Updated